depdive/README.md

```diff
@@ -0,0 +1,57 @@
+# Depdive
+
+[![Documentation (master)](https://img.shields.io/badge/docs-master-59f)](https://github.com/diem/whackadep/depdive)
```

Comment:
There is no documentation generated at that link. Assuming we publish, you should provide links to the crate documentation on docs.rs (and a link to crates.io) using links that will exist once we publish.